recipe/build.sh

```diff
@@ -2,8 +2,18 @@
 
 set -ex
 
+if [[ "$(uname)" == "Linux" && "${target_platform}" != "${build_platform}" ]];
+then
+    echo "ERROR: cross-compilation is not supported; we tried, but couldn't complete it (https://github.com/conda-forge/tensorflow-feedstock/pull/426)"
+    exit 1
+fi    
+
+
+NPROC=$(nproc)
 if [[ "$CI" == "github_actions" ]]; then
-  export CPU_COUNT=4
+    export CPU_COUNT=4
+elif [[ $((NPROC>(CPU_COUNT*2))) ]]; then
+    export CPU_COUNT=$((NPROC))
```

Comment:
Please remove this. you can set the environment variable in your local builds and it will get passed on.